rivet.sh

```diff
@@ -1,76 +1,147 @@
 package: Rivet
 version: "%(tag_basename)s"
-tag: "3.1.6-alice1"
-source: https://github.com/alisw/rivet
+tag: "rivet-3.1.8"
+source: https://gitlab.com/hepcedar/rivet.git
```

Comment:
Some modifications were done in the past to make RIVET work on the ALICE framework, I fear this force pull from the hepcedar repo will break some stuff. @maireiphc is definitely more informed than me on this